examples/kubernetes/vtgate-service.yaml

```diff
@@ -7,7 +7,10 @@ metadata:
     app: vitess
 spec:
   ports:
-    - port: 15001
+    - name: web+bsonrpc
```

Comment:
I did test this out, this is an invalid port name:
spec.ports[0].name: invalid value 'web+bsonrpc', Details: must be a DNS label (at most 63 characters, matching regex [a-z0-9]([-a-z0-9]*[a-z0-9])?): e.g. "my-name"